### ISSUE TITLE
Duplicate hp bar for stunned enemies

### ISSUE THREAD

**Issue by Mmlh1** [cigumo/krvi#222]
https://cdn.discordapp.com/attachments/493772773081219073/763451284744306729/Kingdom_Rush_Vengeance_2020-10-07_19-07-47_Trim.mp4

Enemy hp bars sway back and forth when they are stunned. However, the original, non-moving one is also still there, so they overlap.

**Comment by sunnydaler** [comment]
I wud prefer hp bar not to swing along with the body if easy to correct

**Comment by pablorealini** [comment]
I think this is just a visual stuff....there are no duplicated health bars.
Also it will be very hard to not move the hp bar with the body, since the bar is inside the unit.

**Comment by Mmlh1** [comment]
Yeah I meant it was a small visual glitch but this is so tiny that it's not really something that needs to be fixed.

**Comment by Mmlh1** [comment]
I think that is a better description of the issue. The rotation of the hp bar just makes it look weird.
